### PR TITLE
fix yield running constrain

### PR DESCRIPTION
**Pull request by Aiee**
YIELD now can be run without specifying a space

related issue: http://jira.nebula-graph.io/browse/NG-310

### CODE REVIEW

**Review by jievince — DISMISSED** [review]

**Review by yixinglu — APPROVED** [review]

**Review by yixinglu — COMMENTED** [review]
now you can add some yield test cases in an empty graph

**Review by yixinglu — DISMISSED** [review]

File: tests/tck/features/yield/Yield.feature [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    Given an empty graph
+    When executing query:
+      """
+      YIELD 1+1, '1+1', (int)3.14, (string)(1+1), (string)true
+      """
+    Then the result should be, in any order:
+      | (1+1) | 1+1   | (INT)3.14 | (STRING)(1+1) | (STRING)true |
+      | 2     | "1+1" | 3         | "2"           | "true"       |
+
+  Scenario: [3] yield without chosen space
```
Comment (laura-ding) [inline tests/tck/features/yield/Yield.feature:23]: add test when yield with tag expression

**Review by CPWstatic — COMMENTED** [review]
Please add some test such as `tag.prop`, `label`. Make sure we had a proper error message.

**Review by jievince — DISMISSED** [review]

**Review by yixinglu — DISMISSED** [review]

**Review by CPWstatic — APPROVED** [review]

**Review by yixinglu — DISMISSED** [review]

**Review by nevermore3 — APPROVED** [review]

**Review by CPWstatic — APPROVED** [review]

**Review by yixinglu — APPROVED** [review]
